cmd/gctcli/commands.go

```diff
@@ -3983,7 +3987,7 @@ var getHistoricCandlesCommand = cli.Command{
 		},
 		cli.Int64Flag{
 			Name:        "granularity, g",
-			Usage:       "value is in seconds and can be one of the following {60, 300, 900, 3600, 21600, 86400}",
+			Usage:       "value is in seconds and can be one of the following {60 (1 Minute), 300 (5 Minute), 900 (15 Minute), 3600 (1 Hour), 21600 (6 Hour), 86400 (1 Day)}",
```

Comment:
Would be nice to state these as an example as moving forward other exchanges will support different time durations (4 hours, 1 week etc)